### DISCUSSION TITLE
How to make side menu over non-client area?

### DISCUSSION THREAD

**Discussion by GF-Huang** [HandyOrg/HandyControl#703] [Q&A] [answered]
Like WeChat:

![image](https://user-images.githubusercontent.com/4510984/110346548-a353d100-806a-11eb-8d34-07b2f9374166.png)

**Comment by ghost1372** [comment, marked as answer]
this mode should be implemented in Windows style  
In modernWPF, this feature has been implemented
https://github.com/Kinnara/ModernWpf/blob/2cd256ffe38326d1d4822b5187e2e673fc049990/ModernWpf/TitleBar/TitleBar.cs#L295
I'll try to take a look at it and add this feature if possible😁

**Reply by ghost1372** [reply]
not bad but i need to fix buttons (Min, Max, Close) 😁
![image](https://user-images.githubusercontent.com/9213496/110356944-01bf8600-8050-11eb-96b8-dbfb26756bdb.png)

**Reply by ghost1372** [reply]
Done😅 https://github.com/HandyOrg/HandyControl/pull/704
![image](https://user-images.githubusercontent.com/9213496/110367432-e4dd7f80-805c-11eb-839d-05804cc3183b.png)
